ddtrace/profiling/exporter/pprof.pyx

```diff
@@ -181,14 +181,10 @@ class _PprofConverter(object):
             ),
         )
 
-        nevents = len(events)
-        sampling_ratio_avg = sum(event.capture_pct for event in events) / nevents / 100.0
-        total_alloc = sum(event.nevents for event in events)
-        number_of_alloc = total_alloc * sampling_ratio_avg
-        average_alloc_size = sum(event.size for event in events) / float(nevents)
-
-        self._location_values[location_key]["alloc-samples"] = nevents
-        self._location_values[location_key]["alloc-space"] = round(number_of_alloc * average_alloc_size)
+        self._location_values[location_key]["alloc-samples"] = sum(event.nevents for event in events)
+        self._location_values[location_key]["alloc-space"] = round(
+            sum(event.size / event.capture_pct * 100.0 for event in events)
```

Comment:
❓ Maybe I'm missing something, but shouldn't this be?

```suggestion
            sum(event.size / (event.capture_pct / 100.0) for event in events)
```

Reply:
They are equivalent 🙂

Reply:
Oh lol, of course. Not sure what I was thinking 🤦.